config_template.py

```diff
@@ -71,6 +71,12 @@
 ### Example: urn:oid:2.5.4.4
 #SAML_ATTRIBUTE_SURNAME = 'urn:oid:2.5.4.4'
 
+## Split into Given name and Surname
```

Comment:
Can you also sync the new changes to config files in `configs/`

Reply:
done